app/controllers/v1/statuses_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module V1
+  # API for Compliance Status
+  class StatusesController < ApplicationController
+    skip_around_action :authenticate_user
+
+    def show
+      render json: { data: all_statuses }, status: status_return_code
+    end
+
+    private
+
+    def all_statuses
+      @all_statuses ||= {
+        api: api_status
+      }
+    end
+
+    def status_return_code
+      all_statuses.values.all? ? :ok : :internal_server_error
+    end
+
+    def api_status
+      ActiveRecord::Base.establish_connection
```

Comment:
Restablishing a connection does not feel right to do here. It seems that this call will create a new connection pool.

Reply:
Nope, if the connection is already established, the method just returns false.

Reply:
Are you sure about this? It seems to call `remove_connection` within the pool https://github.com/rails/rails/blob/v5.2.4.5/activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb#L973

Reply:
It seems like calling `establish_connection` does `remove_connection`, so I have added a conditional to only call it if the connection is not already active. Please give it another test.